packages/cli/src/lib/addons/resolve.ts

```diff
@@ -113,7 +116,7 @@ export const attachmentResolver = async (heroku: APIClient, app: string | undefi
     if (attachment) {
       return attachment
     }
-  } catch {}
+  } catch  {}
```

Comment:
Probably should remove this unintentional whitespace addition

Reply:
Applied formatting to the entire file. Done.